src/message.js

```diff
@@ -106,7 +106,11 @@ class Message {
 
     let key = this.conversation_id
     let expiration = Date.now() + secondsToExpire * 1000
-    this._slapp.convoStore.set(key, { fnKey, state, expiration })
+    this._slapp.convoStore.set(key, { fnKey, state, expiration }, (err) => {
+      if (err) {
+        this._slapp.emit('error', err)
```

Comment:
instead of emitting the error shouldn't you just invoke the error function? e.g. `err()`?